proxywasm/v2/exports.go

```diff
@@ -44,15 +44,15 @@ func (a *ABIContext) ProxyOnMemoryAllocate(memorySize int32) (int32, error) {
 }
 
 func (a *ABIContext) ProxyOnContextCreate(contextID int32, parentContextID int32, contextType ContextType) (int32, error) {
-	res, _, err := a.CallWasmFunction("proxy_on_context_create", contextID, parentContextID, contextType)
+	_, _, err := a.CallWasmFunction("proxy_on_context_create", contextID, parentContextID)
 	if err != nil {
 		return 0, err
 	}
-	return res.(int32), nil
+	return 0, nil
 }
 
 func (a *ABIContext) ProxyOnContextFinalize(contextID int32) (int32, error) {
```

Comment:
function name should also keep consistent

Reply:
On the basis of which WasmFunction called, change it like `ProxyOnContextFinalize` to `ProxyOnDone`. Is it?

Reply:
yes

Reply:
ok, I will replace it soon

Reply:
Done